Tests/Pcap++Test/Tests/LiveDeviceTests.cpp

```diff
@@ -248,6 +262,49 @@ PTF_TEST_CASE(TestPcapLiveDevice)
 
 
 
+PTF_TEST_CASE(TestPcapLiveDeviceClone)
```

Comment:
Since this test requires a network connection it won't run in CI. Did you test it on your local machine?

Reply:
Yes, I checked with Ubuntu 20.04 WSL